graf2d/asimage/src/libAfterImage/ungif.c

```diff
@@ -229,8 +230,9 @@ get_gif_saved_images( GifFileType *gif, int subimage, SavedImage **ret, int *ret
 				{
             		/* Create an extension block with our data */
 #if (GIFLIB_MAJOR>=5)
+				      Len = EGifGCBToExtension(gif, ExtData);
```

Comment:
By coincidence this returns `4` in runtime, which should fit in `sizeof(void*)` on most platforms.